evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,403 @@
+            }
+
+            active.attr("data-review", action);
+            updateButtons();
+
+            slide("right");
+            slider.find("[data-action=" + action + "]").focus();
+        }
+
+        function updateState() {
+            slider.find(".slider-side-left").toggleClass("shown", index > 0);
+            let reviewed = items.slice(0, index).filter(item => item.is("[data-review]")).length;
+            slider.find("[data-counter=reviewed-left]").text(reviewed);
+            slider.find("[data-counter=unreviewed-left]").text(index - reviewed);
+
+            slider.find(".slider-side-right").toggleClass("shown", active !== null);
+            reviewed = items.slice(index + 1).filter(item => item.is("[data-review]")).length;
+            slider.find("[data-counter=reviewed-right]").text(reviewed);
+            let unreviewed = items.length - index - reviewed - 1;
+            slider.find("[data-counter=unreviewed-right]").text(Math.max(unreviewed, 0));
+
+            if(!active) {
+                let reviewed = items.every(item => item.is("[data-review]"));
+                let alert = slider.find(".alert");
+                alert.toggleClass("alert-warning", !reviewed);
+                alert.toggleClass("alert-success", reviewed && items.length !== 0);
+                alert.toggleClass("alert-secondary", reviewed && items.length === 0);
+                alert.find("[data-content=unreviewed]").toggleClass("d-none", reviewed);
+                alert.find("[data-content=reviewed]").toggleClass("d-none", !reviewed);
+                slider.find("[data-action=repeat-unreviewed]").toggleClass("d-none", reviewed);
+                slider.find("[data-action=repeat-all]").toggleClass("d-none", items.length === 0);
+            }
+        }
+
+        function updateButtons() {
+            slider.find("[data-action-set=reviewing]").toggleClass("d-none", !active);
+            slider.find("[data-action-set=repeat]").toggleClass("d-none", active !== null);
+
+            if(active) {
+                // focus the answer to enable scrolling with arrow keys
+                active.focus();
+
+                let actions = {
+                    publish: "btn-success",
+                    make_private: "btn-dark",
+                    hide: "btn-danger"
+                };
+
+                let review = active.attr("data-review");
+                for(let action in actions) {
+                    let btn = slider.find("[data-action=" + action + "]");
+                    btn.toggleClass(actions[action], review === action);
+                    btn.toggleClass("btn-outline-secondary", review !== action);
+                }
+
+                let notContributor = !active.data("contribution");
+                let privateBtn = slider.find("[data-action=make_private]");
+                privateBtn.attr("disabled", notContributor);
+                privateBtn.tooltip(notContributor? 'enable' : 'disable');
+            }
```

Comment:
same for `updateButtons`, please check whether there's a reasonable way to split this up.